clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -1403,48 +1423,34 @@ describe("Fides-js TCF", () => {
           experience: OVERRIDE.UNDEFINED,
         },
         OVERRIDE.UNDEFINED,
-        OVERRIDE.UNDEFINED
+        OVERRIDE.EMPTY
```

Comment:
Without this `EMPTY`, I was finding the specs failed sometimes because an unexpected `GET /privacy-experience` was triggered

Reply:
oh weird. I guess EMPTY more accurately reflects the real-world anyway, where we'd still successfully call for client-side experience but it returns an empty response 👍

Reply:
Yeah- for better spec coverage, we could also test for receiving a TCF experience as well and see what happens in that case, but that seemed pretty well covered elsewhere